src/test/java/bio/terra/service/snapshot/flight/SnapshotAuthzIamStepTest.java

```diff
@@ -56,6 +59,11 @@ class SnapshotAuthzIamStepTest {
   void setup() {
     workingMap = new FlightMap();
     snapshotRequestModel = new SnapshotRequestModel();
+    // Set mode to something other than byRequestId
+    snapshotRequestModel.contents(
+        List.of(
```

Comment:
You can use `addContentsItem()` here
```suggestion
    snapshotRequestModel.addContentsItem(
```

Reply:
Since we're setting the snapshot mode in the setup method, this needs to override the snapshot mode. Adding a content item does not override the original mode.